baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/StrictUnusedVariableTest.java

```diff
@@ -145,6 +145,31 @@ public void fails_suppressed_but_used_variables() {
                 "}").doTest();
     }
 
+    @Test
+    public void side_effects_are_preserved() {
+        refactoringTestHelper
+                .addInputLines(
+                        "Test.java",
+                        "class Test {",
+                        "  private static int _field = 1;",
+                        "  public static void privateMethod() {",
+                        "    Object foo = someMethod();",
+                        "  }",
+                        "  private static Object someMethod() { return null; }",
+                        "}")
+                .addOutputLines(
+                        "Test.java",
+                        "class Test {",
+                        "  private static int field = 1;",
+                        "  public static void privateMethod() {",
+                        "    someMethod();",
+                        "  }",
+                        "  private static Object someMethod() { return null; }",
```

Comment:
seems like the test is rewriting this method to be multi-line for some reason...?
or maybe it's comparing AST _first_ and failing on that before comparing text

Reply:
nvm, hadn't noticed the extra `}`